src/printer.js

```diff
@@ -3260,6 +3273,84 @@ function hasLeadingOwnLineComment(text, node) {
   return res;
 }
 
+function hasNakedLeftSide(node) {
+  return (
+    node.type === "BinaryExpression" ||
```

Comment:
No problem. :) I originally commented because you mentioned in another comment that this was a bit ugly. Since that's the convention elsewhere, it makes sense to follow it. :)